### PR TITLE
[ng] src reorg of toh-3 example; prose introduces `lib/src`

### PR DESCRIPTION
**Pull request by chalin**
Contributes to #713

### CODE REVIEW

File: src/angular/tutorial/toh-pt3.md [hunk 1]
```diff
@@ -57,11 +57,13 @@ This file will hold the new `HeroDetailComponent`.
     case_](/angular/glossary#snake_case), each word separated by
     underscores, and end in `_component.dart`.  The `HeroDetailComponent`
```
Comment (kwalrath) [inline src/angular/tutorial/toh-pt3.md]: end -> ends or ending

Comment (chalin) [inline src/angular/tutorial/toh-pt3.md]: Really? If we remove the "each word ...", it currently reads:
> The component _file_ name should be spelled in [_lower underscore case_](/angular/glossary#snake_case), ..., and end in `_component.dart`.

This seems fine to me.

Comment (kwalrath) [inline src/angular/tutorial/toh-pt3.md]: Yeah, you're right, it's fine grammatically, if a little hard to parse. It's a bit redundant, too, and takes you to a definition (snake_case) that isn't mentioned here. Maybe:

The component _file_ name should be in [snake case](/angular/glossary#snake_case)—lowercase with underscore separation—and end in `_component.dart`.

Comment (chalin) [inline src/angular/tutorial/toh-pt3.md]: Yes, that is better. Fixed as you suggested.

File: src/angular/tutorial/toh-pt3.md [hunk 2]
```diff
@@ -49,14 +49,13 @@ This file will hold the new `HeroDetailComponent`.
 <div class="l-sub-section" markdown="1">
   **Angular conventions**:
 
-  * The component _class_ name should be written in _upper camel case_ and
+  * The component _class_ name should be written in [upper camel case](/angular/glossary#pascalcase) and
```
Comment (kwalrath) [inline src/angular/tutorial/toh-pt3.md]: good catch!
